app/models/manager_refresh/inventory_collection.rb

```diff
@@ -73,10 +73,10 @@ class InventoryCollection
     attr_accessor :parent_inventory_collections
 
     attr_reader :model_class, :strategy, :attributes_blacklist, :attributes_whitelist, :custom_save_block, :parent,
-                :internal_attributes, :delete_method, :dependency_attributes, :manager_ref,
+                :internal_attributes, :delete_method, :dependency_attributes, :manager_ref, :create_only,
```

Comment:
When would we want `:create_only`?

Reply:
we can now enforce to do only insert or upsert by this (so we will avoid loading a lot of records)